modules/sdk-coin-eth/src/lib/transactionBuilder.ts

```diff
@@ -236,7 +236,11 @@ export class TransactionBuilder extends BaseTransactionBuilder {
           this.forwarderVersion(1);
           this.baseAddress(baseAddress);
           this.salt(addressCreationSalt);
-          this.initCode((this._coinConfig.network as EthereumNetwork).forwarderImplementationAddress as string);
+          const forwarderImplementationAddress = (this._coinConfig.network as EthereumNetwork)
```

Comment:
BSC coin doesn't need address initialization and forwarders are also not implemented yet. I don't understand this pr. Can you elaborate the goal of this pr?

Reply:
Hey we need bsc txBuilder the ability to build `AddressInitialization`, `WalletInitialization` similar to eth txBuilder.
This is required for cross chain recovery as we need to deploy v0/v1 wallet and forwarder to perform recovery